extensions/jdbc/jdbc-mariadb/deployment/src/main/java/io/quarkus/jdbc/mariadb/deployment/JDBCMariaDBProcessor.java

```diff
@@ -49,28 +47,15 @@ void configureAgroalConnection(BuildProducer<AdditionalBeanBuildItem> additional
         }
     }
 
-    @BuildStep
-    void registerAuthenticationPlugins(BuildProducer<ServiceProviderBuildItem> serviceProvider) {
-        // make sure that all plugins are available
-        serviceProvider
-                .produce(ServiceProviderBuildItem.allProvidersFromClassPath("org.mariadb.jdbc.plugin.AuthenticationPlugin"));
-    }
-
-    @BuildStep(onlyIf = NativeOrNativeSourcesBuild.class)
-    void addNativeImageResources(BuildProducer<NativeImageResourceBuildItem> resources) {
-        // mariadb.properties is used by org.mariadb.jdbc.util.VersionFactory and is small enough.
-        // driver.properties is not added because it only provides optional descriptions for
-        // org.mariadb.jdbc.Driver.getPropertyInfo(), which is probably not even called.
-        resources.produce(new NativeImageResourceBuildItem("mariadb.properties"));
```

Comment:
I have a vague recollection you said we already had a warning because of that before the upgrade?

Reply:
Hum, no - I haven't seen a warning before the upgrade (IIRC). I was merely surprised that there (seemingly) wasn't a problem with that missing file, given that has been used by the client for a long time.
Anyway, I'll double check the CI logs.

Reply:
The logs of the `Data1` run look alright.